packages/styles/components/_c.toggle.scss

```diff
@@ -0,0 +1,101 @@
+.mc-toggle {
+  display: block;
+
+  // hide the input
+  &__input {
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    padding: 0;
+    margin: -1px;
+    overflow: hidden;
+    clip: rect(0, 0, 0, 0);
+    border: 0;
+    visibility: visible;
+    white-space: nowrap;
+  }
+
+  &__label {
+    @include set-font-scale('05', 'xs');
+    @include set-toggle-size('m');
+
+    display: inline-block;
+    white-space: nowrap;
+    cursor: pointer;
+    position: relative;
+    user-select: none;
+
+    :disabled + & {
+      cursor: default;
+    }
+
+    &::after,
+    &::before {
+      box-sizing: border-box;
+      position: absolute;
+      top: 0;
+      left: 0;
+      display: block;
+      content: '';
+    }
+
+    // the switch background
+    &::before {
+      background-color: $color-toggle-off-background;
+      will-change: background-color;
```

Comment:
Is it for the safari bug ?

Reply:
no this is a common property to activate GPU acceleration on transitions. 
It is a overall good practice and should be generalized.